packages/simulation/pitop/simulation/sprites.py

```diff
@@ -199,7 +252,7 @@ def create_components(self):
             step=self.slider_step,
             initial=0,
             handleRadius=int(SLIDER_HANDLE_RADIUS * self.scale),
-            handleColour=pygame.Color("#505050"),
+            handleColour=PitopColor.CHARCOAL.value,
```

Comment:
```suggestion
            handleColour=pygame.Color(PitopColor.CHARCOAL.value),
```

This is actually the reason for the broken/stalling tests. Pytest and xdist make this not at all clear to debug, but you can see the error if you use pytest -s flag (or probably also if you disable xdist):
```
venv ❯ pytest tests/test_pitop.py -s -k test_pitop_simulate
...blahblah
Process Process-1:
Traceback (most recent call last):
  File "/usr/lib/python3.9/multiprocessing/process.py", line 315, in _bootstrap
    self.run()
  File "/usr/lib/python3.9/multiprocessing/process.py", line 108, in run
    self._target(*self._args, **self._kwargs)
  File "/home/angus/pi-top/src/pi-top-Python-SDK/packages/simulation/pitop/simulation/simulation.py", line 148, in _run
    sprite_group = sprite_class.create_sprite_group(size, config, scale)
  File "/home/angus/pi-top/src/pi-top-Python-SDK/packages/simulation/pitop/simulation/simsprite.py", line 83, in create_sprite_group
    sprite = sprite_class(component, scale, draw_port=True)
  File "/home/angus/pi-top/src/pi-top-Python-SDK/packages/simulation/pitop/simulation/sprites.py", line 230, in __init__
    self.image = self.render()
  File "/home/angus/pi-top/src/pi-top-Python-SDK/packages/simulation/pitop/simulation/sprites.py", line 313, in render
    self.slider.draw(image)
  File "/home/angus/pi-top/src/pi-top-Python-SDK/packages/simulation/pitop/simulation/widgets/slider.py", line 123, in draw
    gfxdraw.filled_circle(
TypeError: invalid color argument
```